os_net_config/impl_ifcfg.py

```diff
@@ -1505,6 +1505,33 @@ def generate_route_table_config(self, route_tables):
                         % (id, route_tables[id])
         return data
 
+    def _get_dpdk_pci_addresses(self, ifcfg_dpdk_file):
+        """Parse ifcfg-dpdk file and extract PCI addresses from DPDK devargs.
+
+        :param ifcfg_dpdk_file: ifcfg file for the dpdk port or bond
+        :returns: List of PCI addresses found in options:dpdk-devargs
+        """
+        pci_addresses = []
+
+        devargs_pattern = re.compile(r"options:dpdk-devargs=([0-9a-fA-F:.]+)")
```

Comment:
I'm going to trust that this regex which is pretty broad (matches any letter or number or colon or "." and has the "+") is not going to accidentally match any valid OVS options that follow. I think that's OK, but just noting that it is a gamble.

Reply:
Thanks. Fixed it now